napari/__main__.py

```diff
@@ -6,13 +6,32 @@
 
 import numpy as np
 
+from . import Viewer, __version__, gui_qt
 from .utils import io
-from . import Viewer, gui_qt
+from napari_info import citation_text
+
+
+class CitationAction(argparse.Action):
+    def __call__(self, *args, **kwargs):
+        print(citation_text)
+        sys.exit()
 
 
 def main():
     parser = argparse.ArgumentParser(usage=__doc__)
     parser.add_argument('images', nargs='*', help='Images to view.')
+    parser.add_argument(
+        '-v',
+        '--version',
+        action='version',
+        version=f'napari version {__version__}',
+    )
+    parser.add_argument(
```

Comment:
I think I'd leave in the `napari --info` syntax here too - I prefer it to `napari-info` and see `napari-info` as more of a last resort for people that have install fails and document it as such

Reply:
I agree with this sentiment very much.